internal/olm/installer/client.go

```diff
@@ -196,13 +196,13 @@ func (c Client) getResources(ctx context.Context, version string) ([]unstructure
 			return nil, err
 		}
 	} else {
-		log.Infof("Fetching resources for resolved version %q", version)
-		crdResources, err = c.getCRDs(ctx, version)
+		log.Infof("Fetching resources for resolved version %q", resolvedVersion)
+		crdResources, err = c.getCRDs(ctx, resolvedVersion)
 		if err != nil {
 			return nil, fmt.Errorf("failed to fetch CRDs: %v", err)
 		}
 
-		olmResources, err = c.getOLM(ctx, version)
+		olmResources, err = c.getOLM(ctx, resolvedVersion)
 		if err != nil {
 			return nil, fmt.Errorf("failed to fetch resources: %v", err)
 		}
```

Comment:
Missing update all e2e tests to use 0.17.0 now. See the changes made in the PR: https://github.com/operator-framework/operator-sdk/pull/4056/files